optaplanner-core/src/test/java/org/optaplanner/core/impl/localsearch/decider/forager/finalist/StrategicOscillationByLevelFinalistPodiumTest.java

```diff
@@ -212,4 +213,8 @@ protected LocalSearchMoveScope buildMoveScope(LocalSearchStepScope stepScope,
         return moveScope;
     }
 
+    private static <E> void verifyListContainsExactlyOneInstance(List<E> list, E instance) {
```

Comment:
Just for brevity: `assertThat(list).assertContainsExactly(instance);` serves the same purpose.